cookietemple/info/info.py

```diff
@@ -133,7 +133,7 @@ def output_table(templates_to_print: list, handle: str) -> None:
         table.add_column("Version", justify="left")
 
         for template in templates_to_print:
-            table.add_row(f'[bold]{template[0]}', template[1], template[2], template[3], template[4])
+            table.add_row(f'[bold]{template[0]}', template[1], f'{template[2]}\n', template[3], template[4])
```

Comment:
why?

Reply:
I added a blank line because otherwise it was hardly visible to distinguish between two templates

Reply:
Mhm, I guess I'll have to see this...